rt_gene_model_training/pytorch/utils/GenerateEyePatchesDataset.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function, division, absolute_import
+
+import argparse
+import os
+
+import cv2
+from tqdm import tqdm
+
+from rt_gene.extract_landmarks_method_base import LandmarkMethodBase
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Estimate gaze from images')
+    parser.add_argument('im_path', type=str, default=os.path.join(script_path, '../samples/natural'),
```

Comment:
Use abspath here and in the line below, too.

Reply:
done.